verification/benchmark_user_events_v2.php

```
@@ -0,0 +1,194 @@
+    global $db_queries, $term_cache;
+    if (isset($term_cache[$post_id])) {
+        return $term_cache[$post_id];
+    }
+    $db_queries++;
+    usleep(100); // Simulate DB latency per query
+    // Return mock terms
+    if ($post_id % 2 == 0) { // Simulate some matching products
+        return [(object)['slug' => 'events'], (object)['slug' => 'other']];
+    }
+    return [(object)['slug' => 'clothing']];
+}
```

PHPMD (2.15.0)

[error] The parameter $post_id is not named in camelCase. (undefined)

(CamelCaseParameterName)